hardware.inc

```diff
@@ -384,7 +384,7 @@ DEF AUDVOL_VIN_RIGHT EQU %00001000 ; SO1
 ; -- Bit 3   - Output sound 4 to SO1 terminal
 ; -- Bit 2   - Output sound 3 to SO1 terminal
 ; -- Bit 1   - Output sound 2 to SO1 terminal
-; -- Bit 0   - Output sound 0 to SO1 terminal
+; -- Bit 0   - Output sound 1 to SO1 terminal

```

Comment:
Arguably, "sound" should be changed to "channel". Also, maybe we could finally document which of SO1 and SO2 is left/right? ><"

Reply:
Yes, I agree. Just added these new changes to the pull request.